langchain/tools/base.py

```diff
@@ -123,3 +144,7 @@ async def arun(
                 observation, verbose=verbose_, color=color, name=self.name, **kwargs
             )
         return observation
+
+    def __call__(self, tool_input: str) -> str:
+        """Make tools callable with str input."""
```

Comment:
This docstring isn't strictly true, as it will actually work to call this with a dict